src/covid_model_seiir_pipeline/forecasting/data.py

```diff
@@ -21,7 +21,9 @@ def __init__(self,
 
     @classmethod
     def from_specification(cls, specification: ForecastSpecification) -> 'ForecastDataInterface':
-        forecast_paths = paths.ForecastPaths(specification.data.output_root, read_only=False)
+        forecast_paths = paths.ForecastPaths(specification.data.output_root,
+                                             read_only=False,
+                                             scenarios=list(specification.scenarios))
```

Comment:
What type are `scenarios` if not a list?

Reply:
Dict[str, ScenarioSpecification]